ui/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter.js

```diff
@@ -100,6 +112,7 @@ class Filter extends React.Component {
           includedAttributes={this.props.includedAttributes}
           editing={this.props.editing}
           onChange={this.updateAttribute}
+          settingsModel={this.getListofSupportedAttributes(this.props.settingsModel)}
```

Comment:
```suggestion
          settingsModel={this.getListOfSupportedAttributes(
            this.props.settingsModel
          )}
```


brought to you by `/projects/ddf/ui>> yarn format -w`